Сопроводиловка

dlipeev opened the pull request:
Привет. 
Есть вопрос по логике работы с МоимСкладом
Дело в том, что для каждого товара есть "модификация", например цвет или размер. Соответственно, по-разному будет строиться получение остатков.
Вот раздел в документации http://wiki.moysklad.ru/wiki/%D0%92%D1%8B%D0%B3%D1%80%D1%83%D0%B7%D0%BA%D0%B0_%D0%BE%D1%81%D1%82%D0%B0%D1%82%D0%BA%D0%BE%D0%B2
Вопрос: какой из видов получения остатков делать в первую очередь?

Commit:
...

olegnekludov commented:
Дима, напиши, пожалуйста, варианты ответа на твой вопрос (на будущее предлагаю взять это за правило: задавая вопрос, предлагай сразу варианты ответов - желательно не менее трех)

Commit:
разбор xml-ника с информацией о товаре и unit-тест этой функциональности

Commit:
парсинг и тестирование для GoodFolder

Commit:
добавление комментов

Commit:
http://trac.beta2.ru/beta2dev/ticket/25
 объектная модель на основании xsd для движений по поставщику и покупателю

 Документы:
 Заказ покупателя
 Отгрузка
 Возврат покупателя
 Заказ поставщику
 Возврат поставщику
 Приёмка

Commit:
в процессе настройки парсинга по тикету 26
документы:
Заказ покупателя
Отгрузка
Возврат покупателя
Заказ поставщику
Возврат поставщику
Приёмка

Commit:
в процессе тестирования парсинга

Commit:
парсинг и тестирование документов
Заказ покупателя
Отгрузка
Возврат покупателя
Заказ поставщику
Возврат поставщику
Приёмка

Commit:
сделан парсинг для товара и группы товаров

Commit:
#26

dlipeev commented:
по тикету 26
